index.js

```diff
@@ -0,0 +1,72 @@
+const formElement = document.getElementById('form');
+
+formElement.addEventListener("submit", buttonClick);
+
+const calculate = range();
+
+function buttonClick(event) {
+  const firstNumberElement = document.getElementById('firstNumber');
+  const secondNumberElement = document.getElementById('secondNumber');
+  const resultElement = document.getElementById('result');
+  const sourceElement = document.getElementById('source');
+  event.preventDefault();
+  const check = checkNumbers(firstNumberElement, secondNumberElement);
+  if (check !== '') {
+    showError(check, resultElement, sourceElement);
+  } else {
+    const result = calculate(firstNumberElement.value, secondNumberElement.value); // get the sum of the row
+    showResult(result, resultElement, sourceElement);
+  }
+}
+
+function checkNumber(number) {
+  const numberTemplate = /^[1-9]\d*$/;
+  if (!number.value.match(numberTemplate)) {
+    return false;
+  }
+  return true;
```

Comment:
This condition is also redundant and you can write `return !number.value.match(numberTemplate)`

Reply:
Done